src/Commands/Files/GetFileVersion.cs

```diff
@@ -13,6 +13,9 @@ public class GetFileVersion : PnPWebCmdlet
         [Parameter(Mandatory = true)]
         public string Url;
 
+        [Parameter(Mandatory = false)]
+        public SwitchParameter UseVersionExpirationReport;
```

Comment:
Please document this addition in the documentation of [Get-PnPFileVersion](https://github.com/pnp/powershell/blob/dev/documentation/Get-PnPFileVersion.md)